src/decNumber/decBasic.c

```diff
@@ -113,7 +113,7 @@ static decFloat * decCanonical(decFloat *result, const decFloat *df) {
   #endif
   } // block
 
-  // Loop to repair a non-canonical coefficent, as needed
+  // Loop to repair a non-canonical coefficient, as needed

```

Comment:
Do not edit, this is an external library.

Reply:
I'll drop all of these, but notably fa3733cc4c001106847d2591f00abd401ce3e940 changed these files...